src/thor/bidirectional_astar.cc

```diff
@@ -551,6 +562,10 @@ BidirectionalAStar::GetBestPath(valhalla::Location& origin,
   travel_type_ = costing_->travel_type();
   access_mode_ = costing_->access_mode();
 
+  desired_paths_count_ = 1;
+  if (options.has_alternates() && options.alternates())
+    desired_paths_count_ += options.alternates();
```

Comment:
Do we need to protect against bad values here or is `options.alternates()` sanitized upstream of this call?

Reply:
its sanitized in loki